verticapy/connect.py

```diff
@@ -186,71 +247,62 @@ def new_auto_connection(dsn: dict, name: str = "DSN"):
     except:
         pass
```

Comment:
I think it would be better to error out instead of pass.

Reply:
I did a new PR to remove all the "try" in the connect.py file.